event_v2.go

```diff
@@ -0,0 +1,68 @@
+package pagerduty
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// Event includes the incident/alert details
+type V2Event struct {
+	RoutingKey string        `json:"routing_key"`
+	Action     string        `json:"event_action"`
+	DedupKey   string        `json:"dedup_key,omitempty"`
+	Images     []interface{} `json:"images,omitempty"`
+	Client     string        `json:"client,omitempty"`
+	ClientURL  string        `json:"client_url,omitempty"`
+	Payload    *V2Payload    `json:"payload,omitempty"`
+}
+
+// Payload represents the individual event details for an event
+type V2Payload struct {
+	Summary   string      `json:"summary"`
+	Source    string      `json:"source"`
+	Severity  string      `json:"severity"`
+	Timestamp string      `json:"timestamp,omitempty"`
+	Component string      `json:"component,omitempty"`
+	Group     string      `json:"group,omitempty"`
+	Class     string      `json:"class,omitempty"`
+	Details   interface{} `json:"custom_details,omitempty"`
+}
+
+// Response is the json response body for an event
+type V2EventResponse struct {
```

Comment:
Based on our [current API documentation for the Events API v2](https://v2.developer.pagerduty.com/docs/events-api-v2), the response format is different than what is expected here.